pom.xml

```diff
@@ -393,6 +400,36 @@
 
     </dependencyManagement>
 
+    <dependencies>
+        <dependency>
+            <groupId>io.opentelemetry</groupId>
+            <artifactId>opentelemetry-extension-noop-api</artifactId>
+            <version>1.7.0-alpha</version>
+        </dependency>
+        <dependency>
```

Comment:
Even regardless of my comment regarding Java 6/8, this should have been placed in submodule's `pom.xml`, not the root-level one.